Charter.md

```diff
@@ -148,45 +149,57 @@ Meetings may be restricted to Community Group participants, but a public
-due consideration of different opinions, a decision should be publicly
-recorded (where GitHub is used as the resolution of an Issue).
-
-If substantial disagreement remains (e.g. the group is divided) and the
-group needs to decide an Issue in order to continue to make progress,
-the Committers will choose an alternative that had substantial support
-(with a vote of Committers if necessary). Individuals who disagree with
-the choice are strongly encouraged to take ownership of their objection
-by taking ownership of an alternative fork. This is explicitly allowed
-(and preferred to blocking progress) with a goal of letting
-implementation experience inform which spec is ultimately chosen by the
-group to move ahead with.
-
-Any decisions reached at any meeting are tentative and should be
-recorded in a GitHub Issue for groups that use GitHub and otherwise on
-the group\'s public mail list. Any group participant may object to a
-decision reached at an online or in-person meeting within 7 days of
-publication of the decision provided that they include clear technical
-reasons for their objection. The Chairs will facilitate discussion to
-try to resolve the objection according to this decision process.
+## Decision Process
+
+This group will seek to make decisions where there is consensus, as described in this section.
+
+### Work item owner
+
+* Every work item (crate or report) worked on by the Community Group will have a list of owners which will always include the Chairs.
+* This list of owners will be recorded using GitHub's [CODEOWNERS] mechanism.
+* Only the Chairs are the owners of the CODEOWNERS file.
+* Any Community Group participant can ask to be promoted owner of a work item once a pull-request they have submitted on this work item is merged
+  (this rule is loosely inspired by the [pull-request hack]).
+* The Chairs may demote an owner of a work item who has not contributed (with pull-requests or reviews) to that item in the past year.
+
+### New work item
+
+* Any Community Group participant may propose a new work item (crate or report) by making a pull-request on the Community Group's GitHub repository
+  with the label `new-work-item`, which constitutes a call for consensus.
+* No less than two weeks after the pull-request was posted,
+  the Chairs assess consensus based on the feedback on the pull-request,
+  and record a group decision on GitHub, following [Section 5.2] of the W3C process.
+* If the decision is to accept the new work item, the Chairs merge the pull-request and promote its author owner of the new work item.
+* Regardless of the consensus of its participant, the Community Group can not accept a work item that is not in its [scope](#scope-of-work),
+  as defined by this charter.
+
+### Merging pull-requests
+
+* Every Community Group participant will have [write access] to the group's GitHub repository,
+  but merging a pull-request will require approval by at least two owners of the affected work item.
```

Comment:
Does it includes the developer ? Does an Owner is the developer ?

Reply:
@MarcAntoine-Arnaud to make things clearer (hopefully), I added a link on the term "owner", explicitly referring to the section above "Work item owners".

As defined by that section, the owners of a work item are initially the chairs and the person having proposed this work item, but are then augmented by anyone with an accepted PR on this work item. So yes, basically, any active developer of the work item is considered an owner.